docs/sources/reference/components/otelcol.auth.bearer.md

```diff
@@ -38,6 +38,21 @@ Name     | Type     | Description                                      | Default
 When sending the token, the value of `scheme` is prepended to the `token` value.
 The string is then sent out as either a header (in case of HTTP) or as metadata (in case of gRPC).
 
+## Blocks
+
+The following blocks are supported inside the definition of
+`otelcol.auth.bearer`:
+
+Hierarchy | Block      | Description                          | Required
+----------|------------|--------------------------------------|---------
+debug_metrics                        | [debug_metrics][] | Configures the metrics that this component generates to monitor its state. | no
```

Comment:
```suggestion
debug_metrics | [debug_metrics][] | Configures the metrics that this component generates to monitor its state. | no
```
Would you mind doing a search and replace for this please? It's ok to put one space in every table, even if it's not consistent with the other rows in the table.

Reply:
Done! :D